### ISSUE TITLE
DX: use 'better' button labels

### ISSUE THREAD

**Issue by staabm** [desktop/desktop#8591]
### Describe the feature or problem you’d like to solve

github desktop uses a lot of dialogs... most of them look really the same so its not so easy to realize when you as a user e.g. opened "Delete repository" instead of "Delete branch" and therefore confirm  the "Delete repository" operation even if you actually just wanted to delete a branch

dialogs in question (example):

Delete Repo
![grafik](https://user-images.githubusercontent.com/120441/68304382-ee8c5700-00a5-11ea-9f01-468ea2621fa0.png)

Delete Branch
![grafik](https://user-images.githubusercontent.com/120441/68304364-e6341c00-00a5-11ea-927e-95aaddb579d4.png)

suggestion: could the "remove" button in this dialog be more descriptive, e.g. "Remove repository" / "Remove branch" as a label on the button instead of just "Remove".

**Comment by Rexogamer** [comment]
Is it just these two dialogs that you find harder to use, or are there other examples?

**Comment by staabm** [comment]
These are the 2 which I mixed up today, therefore I created the issue. There might be more, but none which came into my way yet

**Comment by staabm** [comment]
Just skimmed thru the remaining dialogs.

These look a bit generic:

- https://github.com/desktop/desktop/blob/31c033eea2bbd26efc7752d8543e12ba6ce011e9/app/src/ui/preferences/preferences.tsx#L326
- https://github.com/desktop/desktop/blob/69eed1579caa4183eb952bd14cb82bf9206399fa/app/src/ui/no-repositories/create-tutorial-repository-dialog.tsx#L296

**Comment by outofambit** [comment]
Hi @staabm thanks for filing this issue. I'm sorry these dialogs are resulting in some unwanted behavior.

I understand the problem, but I don't think adding words to the buttons may be sufficient. I would instead suggest some imagery, which could be as simple as https://octicons.github.com/icon/repo/ and https://octicons.github.com/icon/git-branch/ to differentiate branch vs repo operations. This would of course require a bit more design work but perhaps that could help alleviate this.

**Comment by Rexogamer** [comment]
Would octicons render in the buttons or not at the moment?

**Comment by staabm** [comment]
> but I don't think adding words to the buttons may be sufficient

I dont agree. At the moment a button is clicked the focus of the reader is on it and a more descriptive text will help to make sure the user realizes that he/she is in the wrong dialog.

> I would instead suggest some imagery

I agree that an additional image could clear things up

**Comment by staabm** [comment]
today again, I confused the menu entry and the confirm dialog wasn't helpfull, so I deleted the repo, while I wanted to delete a branch
